packages/myst-to-tex/tests/basic.yml

```
@@ -265,3 +265,50 @@ cases:
                   value: my-example
     latex: |-
       my link: \href{https://example.com\#\^my\_example}{my-example}
+  - title: code-block-default
+    mdast:
+      type: root
+      children:
+        - type: mystDirective
+          name: code-block
+          args: python
+          class: my-class
+          value: |-
+            # here is math
+            1+2
+          children:
+            - type: code
+              lang: python
+              class: my-class
+              value: |-
+                # here is math
+                1+2
+    latex: |-
+      \begin{verbatim}
+      # here is math
+      1+2
+      \end{verbatim}
+  - title: code-block-minted
+    mdast:
+      type: root
+      children:
+        - type: mystDirective
+          name: code-block
+          args: python
+          class: minted
```

We could update the test a tiny bit here just to add a new class name:

```suggestion
          class: some-class minted
```